test/helpers/browser.js

```diff
@@ -2,7 +2,9 @@
 // Node module: loopback-boot
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
+/*eslint-env es6*/
```

Comment:
@davidcheung Yes, I agree with eslint env es6 -- @bajtos Already added it specifically to some projects, but we should upgrade eslint-config-loopback like you said.